src/shims/shims.proj

```diff
@@ -2,6 +2,35 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="dir.props" />
 
+  <PropertyGroup>
+    <PackageConfigurations>
```

Comment:
Any reason why you didn't do this in dir.props?

Reply:
Because dir.props is shared with ApiCompat.proj in this case and it was failing when building for 2.0 -- since that was out of the scope in this fix I did it this way, but I can take a look to the failure if needed.